code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -994,13 +987,11 @@ \subsubsection{Data Definitions}
               \item{$\mathbf{v}$ is the velocity ($\frac{\text{m}}{\text{s}}$)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & The kinetic energy of an object is the energy it possess due to its motion.
-        
-        \hyperref[assumpOT]{A: objectTy}
+Notes & Kinetic energy is the measure of the energy a body possesses due to its motion.
         
-        \hyperref[assumpOD]{A: objectDimension}
+        All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
         
-        \hyperref[assumpDI]{A: dampingInvolvement}
+        No damping occurs during the simulation (from \hyperref[assumpDI]{A: dampingInvolvement}).
```

Comment:
We should keep damping here, to remind us that the energy balance changes if energy is converted into heat through friction.  Technically this would come up somewhere else, but that can wait for the future overhaul of this example.